tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/PcmModelsCreatorHelper.java

```diff
@@ -0,0 +1,20 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter.util;
```

Comment:
Should this class not rather reside in the test fragment?

Reply:
Isn't org.palladiosimulator.simexp.dsl.smodel.interpreter.tests the test fragment?

Reply:
yes

Reply:
But I can see only the package...